--- acng.conf.old	2020-04-18 23:19:07.197698201 -0700
+++ acng.conf	2020-04-18 23:12:19.518170190 -0700
@@ -384,7 +384,7 @@
 # the server name with port and must be correctly formated and terminated.
 # Examples:
 # PassThroughPattern: private-ppa\.launchpad\.net:443$
-# PassThroughPattern: .* # this would allow CONNECT to everything
+PassThroughPattern: .* # this would allow CONNECT to everything

 # It's possible that an evil client requests a volatile file but does not
 # retrieve the response and keeps the connection effectively stuck over
